Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scenes/SampleMain.unity

```diff
@@ -640,6 +640,10 @@ PrefabInstance:
       propertyPath: m_Name
```

Comment:
are changes to this scene only serialized values for contract addresses and other previously goerli values?

Reply:
Yes, as well as grouping the scripts in the root for ease of access & adding some new buttons to the sample scene for functions that were not present like contract call, mint 20/721/1155 etc.